app-connector-client/Makefile

```diff
@@ -18,15 +10,14 @@ docker-build:
 	docker build -t $(APP_NAME):latest .
 
 docker-push:
-	docker tag $(APP_NAME) $(IMG):latest
-	docker push $(IMG):latest
-
-npm-publish:
-	npm version "patch" --git-tag-version=false
-	npm publish
+	make docker-build

```

Comment:
Maybe you should call docker build always centrally as it is listed in all sub-makefiles